mindmeld/active_learning/classifiers.py

```diff
@@ -100,6 +101,7 @@ def __init__(
         self,
         app_path: str,
         tuning_level: str,
+        entity_tuning: str,
```

Comment:
Perhaps this should be bool instead of str, as it is a bool in the __init__() method?